app/controllers/reverse_dependencies_controller.rb

```diff
@@ -9,6 +9,8 @@ def index
     @reverse_dependencies = @rubygem.reverse_dependencies
       .by_downloads
       .preload(:gem_download, :latest_version)
+      .distinct(:id)
```

Comment:
Why is `:id` needed? Seems `distinct` supports only true/false arguments.

Reply:
You are right, fixed